test/test-server/networking_handlers.go

```diff
@@ -593,3 +595,40 @@ func handleNetworkingTransitGatewayAttachmentDelete(_ *testing.T, id string) htt
 		}
 	}
 }
+
+func handleNetworkingTransitGatewayAttachmentCreate(t *testing.T) http.HandlerFunc {
```

Comment:
can we add a test case for error response from networking service

Reply:
Added a few cases for error response